kafka/common.go

```diff
@@ -178,9 +190,16 @@ func (cfg *CommonConfig) newClient(additionalOpts ...kgo.Opt) (*kgo.Client, erro
 	if !cfg.DisableTelemetry {
 		kotelService := kotel.NewKotel(
 			kotel.WithTracer(kotel.NewTracer(kotel.TracerProvider(cfg.tracerProvider()))),
-			kotel.WithMeter(kotel.NewMeter()),
+			kotel.WithMeter(kotel.NewMeter(kotel.MeterProvider(cfg.meterProvider()))),
+		)
+		metricHooks, err := NewKgoHooks(cfg.meterProvider())
+		if err != nil {
+			return nil, fmt.Errorf("cannot setup kgo metrics hooks: %w", err)
```

Comment:
For consistency with other errors:
```suggestion
			return nil, fmt.Errorf("kafka: failed creating kgo metrics hooks: %w", err)
```